thunder/core/jit_ext.py

```diff
@@ -1671,7 +1634,7 @@ def is_variableified_tensorproxy(v: Variable | Proxy) -> Proxy:
 
             prim(*args)
 
-        cache_info = thunder._get_cache_info()
+        cache_info = _get_cache_info()
```

Comment:
What's the advantage here?
To my mind, the original version is clearer because you immediately see that it is a function from a different module (and which). I can see how the shorter way is important for things that get used a lot, but this seems distributing the information that this is a utility function from the `thunder/__init__.py` int two lines that are 300 lines apart.